projects/ngx-extended-pdf-viewer/assets/viewer-4.5.732.mjs

```diff
@@ -19334,7 +19544,7 @@ class AnnotationEditorLayer {
   #boundPointerdown = null;
   #boundTextLayerPointerDown = null;
   #editorFocusTimeoutId = null;
-  #editors = new Map();
```

Comment:
why is this renamed?

Reply:
To make it accessible in the mo-pdf-viewer, I need to remove the #, which will make it a public property.

Reply:
You should add accessor methods instead as these can change, and we want to keep the edits minimal.